src/OpenDiffix.Core/Analyzer.fs

```diff
@@ -55,6 +55,8 @@ let private mapFunctionExpression rangeColumns fn parsedArgs =
      AggregateFunction(CountNoise, aggregateArgs), []
    | AggregateFunction (Avg, aggregateArgs), parsedArgs -> //
      mapAvg aggregateArgs parsedArgs Sum Count
+   | AggregateFunction (AvgNoise, aggregateArgs), parsedArgs -> //
+     mapAvg aggregateArgs parsedArgs SumNoise Count
```

Comment:
Why do we have these functions twice (AvgNoise vs DiffixAvgNoise)?

Reply:
I see the same thing with `CountNoise` and `DiffixCountNoise`. The former should not exist because it is not a standard aggregate and there is no implementation for it.

Reply:
So this is to allow specifying AIDs manually like we do for desktop?

Reply:
Ok, gotcha. Works for me.